### DISCUSSION TITLE
4 broken sensors Freestyle Libre 3 in a row

### DISCUSSION THREAD

**Discussion by cherynka** [NightscoutFoundation/xDrip#3388] [Q&A]
Hallo!
My son has been using Freestyle Libre 3 for about a year. The total experience of using Freestyle Libre is 5 years.
We launch the Libre 3 App sensor. For data transfer, we use Juglyuco, Xdrip+, Nightscout applications, the final recipient of data is AAPS. Jugluco also sends data to LibreLinkApp, as far as I understand.
Phone Poco F4
In the last few months we have a big problem with constant data loss. Many times a day my son have to turn bluetooth on and off, or even completely reboot the phone to restore data transmission. Now Turbo mode is turned on. It's gotten better, but there's still a lot of loss.
But the worst thing is that the last 4 sensors in a row worked less than their term. The sensor simply stops transmitting data, and when you try to scan it with your phone, the NFC does not even vibrate, as if there is no sensor on the hand. Tried reading with several phones.
I have already had two sensors replaced by insurance, but they ask questions about error data code from Libre 3 App, and I don't have that, because we only run the sensor with the Libre application. Therefore, for the next 2 sensors, I am afraid to even call them.
Questions:
1. What could be the reason for such massive failure of sensors? Sensors from different batches received at different times. Is it possible that Turbo mode in Jugluco destroying sensors? Or any else app?
2. How can you find an answer to the question of insurance? What error code I need to say? Or (for the future) how to set up data transfer in AAPS to use the native application and display both data and errors?
3. What can be done for more stable data transmission from the sensor?

**Comment by PhilTU** [comment]
I will speculate a little -

1. If unreliable communication is leading to a lot of extra activity perhaps the battery is running down in the sensor. This might explain why you don't get an NFC response either. Power failure. If Abbott took the sensor back and tested it they would find that out, or maybe an electronics hobbyist can open up the failed sensor and determine the state of the battery.

2. If you used the Libre 3 app to connect to the sensor and send data to Libreview then there is the option of using "web follow" data source in xDrip+ to collect that data over the internet and feed into AAPS. The Libre 3 app would give the error codes. This method is used by others (who don't use Juggluco) but obviously requires internet data access and Abbott's servers to work.

3. If you're using juggluco as the primary connector then that's a question to address to the juggluco developer, there is on his web page juggluco.nl a section about troubleshooting and how to send a log. In general Bluetooth works better with fewer signals around and less activity so avoiding headphones, watches and the like *may* have an influence. xDrip+ cannot connect direct to Libre 3 sensor.

**Reply by Kustomwireless** [reply]
Why is it that Xdrip+ can NOT connect to Libre 3 It connects to so many other devices???

**Reply by PhilTU** [reply]
That's an interesting question. I don't know what the stated objectives or scope of the project are. It does not currently work on Libre 2 in US/Canada/Australia or Libre 3 anywhere. This may just reflect availability / enthusiasm of Libre specific developers.
